<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="3" total="6" passed="3">
  <reporter-output>
  </reporter-output>
  <suite name="Calc Suite" duration-ms="0" started-at="2015-07-15T05:06:42Z" finished-at="2015-07-15T05:06:42Z">
    <groups>
    </groups>
  </suite> <!-- Calc Suite -->
  <suite name="SuiteTwo" duration-ms="33256" started-at="2015-07-15T05:06:08Z" finished-at="2015-07-15T05:06:41Z">
    <groups>
    </groups>
    <test name="SuiteTwoCaseOne" duration-ms="17755" started-at="2015-07-15T05:06:08Z" finished-at="2015-07-15T05:06:26Z">
      <class name="SuiteTwo.SuiteTwoCaseOne">
        <test-method status="PASS" signature="checkCaseToRun()[pri:0, instance:SuiteTwo.SuiteTwoCaseOne@5f936fc6]" name="checkCaseToRun" is-config="true" duration-ms="51" started-at="2015-07-15T10:36:08Z" finished-at="2015-07-15T10:36:08Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkCaseToRun -->
        <test-method status="PASS" signature="SuiteTwoCaseOneTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:SuiteTwo.SuiteTwoCaseOne@5f936fc6]" name="SuiteTwoCaseOneTest" duration-ms="14232" started-at="2015-07-15T10:36:08Z" data-provider="SuiteTwoCaseOneData" finished-at="2015-07-15T10:36:22Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[9]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[5]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[45]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SuiteTwoCaseOneTest -->
        <test-method status="PASS" signature="reporterDataResults()[pri:0, instance:SuiteTwo.SuiteTwoCaseOne@5f936fc6]" name="reporterDataResults" is-config="true" duration-ms="6" started-at="2015-07-15T10:36:23Z" finished-at="2015-07-15T10:36:23Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- reporterDataResults -->
        <test-method status="PASS" signature="SuiteTwoCaseOneTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:SuiteTwo.SuiteTwoCaseOne@5f936fc6]" name="SuiteTwoCaseOneTest" duration-ms="2725" started-at="2015-07-15T10:36:23Z" data-provider="SuiteTwoCaseOneData" finished-at="2015-07-15T10:36:26Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[3]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[7]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[21]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SuiteTwoCaseOneTest -->
        <test-method status="PASS" signature="reporterDataResults()[pri:0, instance:SuiteTwo.SuiteTwoCaseOne@5f936fc6]" name="reporterDataResults" is-config="true" duration-ms="7" started-at="2015-07-15T10:36:26Z" finished-at="2015-07-15T10:36:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- reporterDataResults -->
        <test-method status="PASS" signature="closeBrowser()[pri:0, instance:SuiteTwo.SuiteTwoCaseOne@5f936fc6]" name="closeBrowser" is-config="true" duration-ms="16" started-at="2015-07-15T10:36:26Z" finished-at="2015-07-15T10:36:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
      </class> <!-- SuiteTwo.SuiteTwoCaseOne -->
    </test> <!-- SuiteTwoCaseOne -->
    <test name="SuiteTwoCaseTwo" duration-ms="15497" started-at="2015-07-15T05:06:26Z" finished-at="2015-07-15T05:06:41Z">
      <class name="SuiteTwo.SuiteTwoCaseTwo">
        <test-method status="PASS" signature="checkSuiteToRun()[pri:0, instance:SuiteTwo.SuiteTwoCaseTwo@b454096]" name="checkSuiteToRun" is-config="true" duration-ms="95" started-at="2015-07-15T10:36:08Z" finished-at="2015-07-15T10:36:08Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkSuiteToRun -->
        <test-method status="PASS" signature="checkCaseToRun()[pri:0, instance:SuiteTwo.SuiteTwoCaseTwo@b454096]" name="checkCaseToRun" is-config="true" duration-ms="23" started-at="2015-07-15T10:36:26Z" finished-at="2015-07-15T10:36:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkCaseToRun -->
        <test-method status="PASS" signature="SuiteTwoCaseTwoTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:SuiteTwo.SuiteTwoCaseTwo@b454096]" name="SuiteTwoCaseTwoTest" duration-ms="12242" started-at="2015-07-15T10:36:26Z" data-provider="SuiteTwoCaseTwoData" finished-at="2015-07-15T10:36:38Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[9]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[3]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[3]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SuiteTwoCaseTwoTest -->
        <test-method status="PASS" signature="reporterDataResults()[pri:0, instance:SuiteTwo.SuiteTwoCaseTwo@b454096]" name="reporterDataResults" is-config="true" duration-ms="7" started-at="2015-07-15T10:36:38Z" finished-at="2015-07-15T10:36:39Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- reporterDataResults -->
        <test-method status="FAIL" signature="SuiteTwoCaseTwoTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:SuiteTwo.SuiteTwoCaseTwo@b454096]" name="SuiteTwoCaseTwoTest" duration-ms="2669" started-at="2015-07-15T10:36:39Z" data-provider="SuiteTwoCaseTwoData" finished-at="2015-07-15T10:36:41Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[4]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[2]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[2]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
ActualResult Value 2.5 And ExpectedResult Value 2.0 Not Match]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
ActualResult Value 2.5 And ExpectedResult Value 2.0 Not Match
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:38)
	at SuiteTwo.SuiteTwoCaseTwo.SuiteTwoCaseTwoTest(SuiteTwoCaseTwo.java:97)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1220)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SuiteTwoCaseTwoTest -->
        <test-method status="PASS" signature="reporterDataResults()[pri:0, instance:SuiteTwo.SuiteTwoCaseTwo@b454096]" name="reporterDataResults" is-config="true" duration-ms="6" started-at="2015-07-15T10:36:41Z" finished-at="2015-07-15T10:36:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- reporterDataResults -->
        <test-method status="PASS" signature="closeBrowser()[pri:0, instance:SuiteTwo.SuiteTwoCaseTwo@b454096]" name="closeBrowser" is-config="true" duration-ms="15" started-at="2015-07-15T10:36:41Z" finished-at="2015-07-15T10:36:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
      </class> <!-- SuiteTwo.SuiteTwoCaseTwo -->
    </test> <!-- SuiteTwoCaseTwo -->
  </suite> <!-- SuiteTwo -->
  <suite name="SuiteOne" duration-ms="69" started-at="2015-07-15T05:06:08Z" finished-at="2015-07-15T05:06:08Z">
    <groups>
    </groups>
    <test name="SuiteOneCaseOne" duration-ms="29" started-at="2015-07-15T05:06:08Z" finished-at="2015-07-15T05:06:08Z">
      <class name="SuiteOne.SuiteOneCaseOne">
        <test-method status="SKIP" signature="checkCaseToRun()[pri:0, instance:SuiteOne.SuiteOneCaseOne@42de7804]" name="checkCaseToRun" is-config="true" duration-ms="0" started-at="2015-07-15T10:36:08Z" finished-at="2015-07-15T10:36:08Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkCaseToRun -->
        <test-method status="FAIL" signature="SuiteOneCaseOneTest(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:SuiteOne.SuiteOneCaseOne@42de7804]" name="SuiteOneCaseOneTest" duration-ms="0" started-at="2015-07-15T10:36:08Z" data-provider="SuiteOneCaseOneData" finished-at="2015-07-15T10:36:08Z">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[java.lang.NullPointerException]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: java.lang.NullPointerException
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:161)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:429)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1383)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1075)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1180)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1220)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
Caused by: java.lang.NullPointerException
	at utility.SuiteUtility.GetTestDataUtility(SuiteUtility.java:22)
	at SuiteOne.SuiteOneCaseOne.SuiteOneCaseOneData(SuiteOneCaseOne.java:160)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:135)
	... 21 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SuiteOneCaseOneTest -->
        <test-method status="SKIP" signature="closeBrowser()[pri:0, instance:SuiteOne.SuiteOneCaseOne@42de7804]" name="closeBrowser" is-config="true" duration-ms="0" started-at="2015-07-15T10:36:08Z" finished-at="2015-07-15T10:36:08Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
      </class> <!-- SuiteOne.SuiteOneCaseOne -->
    </test> <!-- SuiteOneCaseOne -->
    <test name="SuiteOneCaseTwo" duration-ms="11" started-at="2015-07-15T05:06:08Z" finished-at="2015-07-15T05:06:08Z">
      <class name="SuiteOne.SuiteOneCaseTwo">
        <test-method status="SKIP" signature="checkSuiteToRun()[pri:0, instance:SuiteOne.SuiteOneCaseTwo@353180dc]" name="checkSuiteToRun" is-config="true" duration-ms="0" started-at="2015-07-15T10:36:07Z" finished-at="2015-07-15T10:36:07Z">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[SuiteOne's SuiteToRun Flag Is 'N' Or Blank. So Skipping Execution Of SuiteOne]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: SuiteOne's SuiteToRun Flag Is 'N' Or Blank. So Skipping Execution Of SuiteOne
	at SuiteOne.SuiteOneBase.checkSuiteToRun(SuiteOneBase.java:37)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:277)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1220)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkSuiteToRun -->
        <test-method status="SKIP" signature="checkCaseToRun()[pri:0, instance:SuiteOne.SuiteOneCaseTwo@353180dc]" name="checkCaseToRun" is-config="true" duration-ms="0" started-at="2015-07-15T10:36:08Z" finished-at="2015-07-15T10:36:08Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkCaseToRun -->
        <test-method status="FAIL" signature="SuiteOneCaseTwoTest(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:SuiteOne.SuiteOneCaseTwo@353180dc]" name="SuiteOneCaseTwoTest" duration-ms="0" started-at="2015-07-15T10:36:08Z" data-provider="SuiteOneCaseTwoData" finished-at="2015-07-15T10:36:08Z">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[java.lang.NullPointerException]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: java.lang.NullPointerException
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:161)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:429)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1383)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1075)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1180)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1220)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
Caused by: java.lang.NullPointerException
	at utility.SuiteUtility.GetTestDataUtility(SuiteUtility.java:22)
	at SuiteOne.SuiteOneCaseTwo.SuiteOneCaseTwoData(SuiteOneCaseTwo.java:132)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:135)
	... 21 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SuiteOneCaseTwoTest -->
        <test-method status="SKIP" signature="closeBrowser()[pri:0, instance:SuiteOne.SuiteOneCaseTwo@353180dc]" name="closeBrowser" is-config="true" duration-ms="0" started-at="2015-07-15T10:36:08Z" finished-at="2015-07-15T10:36:08Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
      </class> <!-- SuiteOne.SuiteOneCaseTwo -->
    </test> <!-- SuiteOneCaseTwo -->
  </suite> <!-- SuiteOne -->
</testng-results>
